src/main/scala/se/nimsa/sbx/box/BoxPushOps.scala

```diff
@@ -0,0 +1,220 @@
+package se.nimsa.sbx.box
```

Comment:
This is the new streaming push formulation. I will provide a flow-chart of the architecture.